core/admin.py

```diff
@@ -1,3 +1,6 @@
 from django.contrib import admin
+from core.models import Client, Loan, Payment
 
-# Register your models here.
+admin.site.register(Client)
+admin.site.register(Loan)
+admin.site.register(Payment)
```

Comment:
We should add a new line at the end of file
https://lintlyci.github.io/Flake8Rules/rules/W292.html